sdk/storage/CMakeLists.txt

```diff
@@ -138,9 +138,25 @@ set (AZURE_STORAGE_DATALAKE_SOURCE
 add_library(azure-storage-file-datalake ${AZURE_STORAGE_DATALAKE_HEADER} ${AZURE_STORAGE_DATALAKE_SOURCE})
 target_link_libraries(azure-storage-file-datalake azure-storage-blob)
 
+set (AZURE_STORAGE_SHARES_HEADER
```

Comment:
`AZURE_STORAGE_FILE_SHARES_HEADER`?

Reply:
But we are also calling `AZURE_STORAGE_DATALAKE_HEADER`, do you think it's necessary to change it into `AZURE_STORAGE_FILE_DATALAKE_HEADER`?

Reply:
I don't know, whichever you like.

i'm just not okay with `AZURE_STORAGE_SHARES_HEADER`

Reply:
Sure. :)